### PR TITLE
Add support for MKL acceleration

### LINKED ISSUE #318: gpu, accelerated cpu support for STT & VAD
- [x] gpu STT mac
- [ ] gpu silero vad mac ? not sure it works
- [ ] cuda silero vad windows ? not sure it works
- [ ] cuda silero vad linux ? not sure it works
- [ ] cuda STT windows (heard it worked for some people)
- [ ] cuda STT linux (heard it worked for some people)

### PR DESCRIPTION
**Pull request by Neptune650**
/claim #318
/fix #318

This adds MKL support for Linux and Windows - since this is a CPU acceleration feature not many changes are needed.

### CODE REVIEW

File: .github/workflows/release-app.yml [hunk 1]
```diff
@@ -51,18 +51,18 @@ jobs:
       matrix:
         include:
```
Comment (louis030195) [inline .github/workflows/release-app.yml:52]: does this run?

last time someone sent a PR on this I tested and it was crashing (wasted my time)

Comment (Neptune650) [reply .github/workflows/release-app.yml:52]: It did run for me yes

**Comment by Neptune650** [comment]
> where can i see the github action that successfully ran?

@louis030195 Here it is: https://github.com/mediar-ai/screenpipe/actions/runs/11527768513/job/32093982159
As you can see it completed successfully.

**Comment by louis030195** [comment]
image
i don't see feature mkl

nor linux

**Comment by Neptune650** [comment]
> image
> i don't see feature mkl
> 
> nor linux

@louis030195 Here is Linux https://github.com/mediar-ai/screenpipe/actions/runs/11527768522/job/32093982118
Also the change can be seen when building a release app.

**Comment by Neptune650** [comment]
@louis030195 
image
Look, here's my local GitHub Actions Release App.

**Comment by Neptune650** [comment]
Here is for macOS
image
It has metal, beta and MKL as described.

**Comment by Neptune650** [comment]
@louis030195 It should be good to merge now, but it's important to note that Intel never supported MKL for macOS M1.

**Comment by louis030195** [comment]
@Neptune650 yeah i meant for linux and windows, macos uses metal anyway

i guess next step would be to make the cuda feature work also

thanks!

/approve

**Comment by louis030195** [comment]
@Neptune650 https://github.com/mediar-ai/screenpipe/actions/runs/11577606392/job/32229398688

**Comment by Neptune650** [comment]
> @Neptune650 https://github.com/mediar-ai/screenpipe/actions/runs/11577606392/job/32229398688
> 
> does not work

@louis030195 It seems the library isn't being installed properly in Linux... I'll fix it.

**Pull request merged**